src/events/crawler/scrapers/USA/ME/index.js

```diff
@@ -0,0 +1,67 @@
+import * as fetch from '../../../lib/fetch.js';
+import * as parse from '../../../lib/parse.js';
+import * as transform from '../../../lib/transform.js';
+import * as geography from '../../../lib/geography.js';
+
+// Set county to this if you only have state data, but this isn't the entire state
+const UNASSIGNED = '(unassigned)';
+
+const scraper = {
+  state: 'ME',
+  country: 'USA',
+  url: 'https://www.maine.gov/dhhs/mecdc/infectious-disease/epi/airborne/coronavirus.shtml',
+  type: 'table',
+  headless: true,
+  aggregate: 'county',
+
+  _counties: [
+    'Androscoggin County',
+    'Aroostook County',
+    'Cumberland County',
+    'Franklin County',
+    'Hancock County',
+    'Kennebec County',
+    'Knox County',
+    'Lincoln County',
+    'Oxford County',
+    'Penobscot County',
+    'Piscataquis County',
+    'Sagadahoc County',
+    'Somerset County',
+    'Waldo County',
+    'Washington County',
+    'York County'
+  ],
+
+  async scraper() {
+    let counties = [];
+    const $ = await fetch.page(this.url);
+    const $th = $('th:contains("Confirmed and Recovered Case Counts by County")');
+    const $table = $th.closest('table');
+
+    const $trs = $table.find('tbody > tr');
+    $trs.each((index, tr) => {
+      const $tr = $(tr);
+      let county = geography.addCounty(parse.string($tr.find('> *:first-child').text()));
+      const cases = parse.number($tr.find('> *:nth-child(2)').text());
+      const recovered = parse.number(parse.string($tr.find('> *:last-child').text()) || 0);
+
+      if (county === 'Unknown County') {
+        county = UNASSIGNED;
+      }
+      if (index < 1) {
```

Comment:
Might want to move this conditional to the top of the function, such that we don't try (and fail) to parse cases/recovered from these rows.